css/components/hero.css

```diff
@@ -0,0 +1,85 @@
+.container-hero {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: #F4EFE9;
+}
+
+.row-hero-text {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-content: center;
+    justify-content: center;
+  }
+
+
+h7.hero {
```

Comment:
Nežinau, kokia yra *geroji praktika* bet `element.class` selektorius man nelabai patinka. Keli elementai su ta pačia klase o visi atrodo skirtingai. Galbūt geriau parasyti unikalias pagal kokią CSS klasių rašymo konvenciją. [Jei domina, paskaityk](https://hackernoon.com/best-practice-in-css-organisation-and-naming-conventions-4d103ujy).
pvz `.row-hero-text__title` arba `.row-hero-text > h7`